azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,281 @@
+
+```Bash
+RES_REGION=westeurope 
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+az cognitiveservices account create \
+-n $ACCT_NAME \
+-g $RES_GROUP \
+-l $RES_REGION \
+--kind ComputerVision \
+--sku S1 \
+ --yes
+```
+
+## Installation
+
+Install the Azure Cognitive Services Computer Vision SDK with [pip][pip], optionally within a [virtual environment][venv].
+
+### Configure a virtual environment (optional)
+
+Although not required, you can keep your base system and Azure SDK environments isolated from one another if you use a virtual environment. Execute the following commands to configure and then enter a virtual environment with [venv][venv]:
+
+```Bash
+python3 -m venv azure-cognitiveservices-vision-computervision-environment
+source azure-cognitiveservices-vision-computervision-environment/bin/activate
+```
+
+### Install the SDK
+
+Install the Azure Cognitive Services Computer Vision SDK for Python with [pip][pip]:
+
+```Bash
+pip install git+https://github.com/johanste/azure-computervision-python.git@ux git+https://github.com/binderjoe/computervision-python-prototype.git@master
+```
+
+## Authentication
+
+To use the Computer Vision SDK, create an instance of the [Computer Vision][ref_computervisionclient]. Once you create your Computer Vision resource, you need its **region**, and one of its **account keys** to instantiate the client object.
+
+### Get credentials
+
+Use the Azure CLI snippet below to populate two environment variables with the Computer Vision account URI and its primary master key (you can also find these values in the Azure portal). The snippet is formatted for the Bash shell.
+
+```Bash
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+export ACCOUNT_REGION=$(az cognitiveservices account show \
+--resource-group $RES_GROUP \
+--name $ACCT_NAME \
+--query location \
+--output tsv)
+
+export ACCOUNT_KEY=$(az cognitiveservices account keys list \
+--name $ACCT_NAME \
+--resource-group $RES_GROUP \
+--name $ACCT_NAME \
+--query key1 \
+--output tsv)
+```
+
+### Create client
+
+Once you've populated the `ACCOUNT_REGION` and `ACCOUNT_KEY` environment variables, you can create the [ComputerVisionAPI][ref_computervisionclient] client.
+
+```Python
+from azure.cognitiveservices.vision.computervision import ComputerVisionAPI
+from azure.cognitiveservices.vision.computervision.models import VisualFeatureTypes
+from msrest.authentication import CognitiveServicesCredentials
+
+import os
+region = os.environ['ACCOUNT_REGION']
+key = os.environ['ACCOUNT_KEY']
+
+credentials = CognitiveServicesCredentials(key)
+client = ComputerVisionAPI(region, credentials)
+```
+
+## Usage
+
+Once you've initialized a [ComputerVisionAPI][ref_computervisionclient] client, you can:
+
+* [Analyze an image][ref_analyzeimage]: You can analyze an internet image for certain features such as faces, colors, tags. The image can be local to the computer, on the internet.  
```

Comment:
"You can analyze an internet image"

If you one can analyze a local image, how about simply "You can analyze an image"?